va/va.go

```diff
@@ -487,7 +498,10 @@ func (va *ValidationAuthorityImpl) checkCAAService(ctx context.Context, ident co
 		if prob.Type == probs.ServerInternalProblem {
 			prob.Detail = "Internal communication failure"
 		} else {
-			prob.Detail = err.Error()
+			// bgrpc.CodeToProblem can return non-gRPC errors passed back by
```

Comment:
Ah, my comment is a bit confusing. I mean't non-internal gRPC errors, i.e. errors returned using non standard gRPC error codes like we do for DNS timeout/error. I need to check where `CodeToProblem` is used but I think it makes sense to use it to also populate the `Detail` field.